app/views/users/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="">
+<%= image_tag @user.photo, width: 100, :class => "rounded mx-auto d-block" %> 
+<h1> <%= @user.name %> </h1>
+</div>
+
+<div>
+<p> <%= @user.bio %></p>
+</div>
+
+<div class="flex flex-col gap-16 h-auto">
+<%@posts.each do |post|%>
+
+<%= link_to posts_path_url do%>
+<div class="post-data">
+        <h2> <%= post.title%> </h2>
+        <p> <%= post.text%> </p>
+       
+</div>
```

Comment:
- [ ] On this page you should consider displaying the number of comments and number of Likes for each post.

Reply:
Changes made 😺